main.go

```diff
@@ -23,6 +23,8 @@ import (
 	"os"
 	"time"
 
+	"github.com/open-policy-agent/gatekeeper/pkg/version"
```

Comment:
nit: can we move this down with the rest of `github.com/open-policy-agent/gatekeeper/..` imports?